docs/guides/options.md

```diff
@@ -374,6 +375,12 @@ Using `<source>` elements will have the same effect:
 </video>
 ```
 
+### `suppressNotSupportedError`
+
+> Type: `boolean`
+
+If set to true, then the no compatible source error will not be triggered immediately and instead will occur on the first user interaction. This is useful for Google's "mobile friendly" test tool, which can't play video but where you might not want to see an error displayed.
```

Comment:
I wonder if there's a way to detect google bot and auto set this

Reply:
They do document the user agents they use but I'm wary about doing that, in case adding special case handling for Googlebot user agents is detected and perceived as an attempt to fiddle SEO.

Reply:
yeah, that make sense.